app/jobs/bulk_update_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class BulkUpdateJob < ApplicationJob
+  def perform(ids:, args:)
+    return unless args[:mark_complete]
```

Comment:
Why would this ever get queued without mark_complete? I mean, I guess it would right now, but wouldn't it be more efficient to just not queue a thing if it didn't need updated?

Reply:
I think this was a sort of breadcrumb for the fact that we will add new options to the form as we go along. but it doesn't really make sense the way it is; I'll look at moving or removing it.